src/app/nav/nav.component.ts

```diff
@@ -10,6 +10,10 @@ export class NavComponent implements OnInit {
 
   constructor() { }
 
+<<<<<<< HEAD
```

Comment:
It looks like there's another merge conflict here to be addressed. This is a great question to post to hacktoberfest channel and to practice resolving.